extmod/vfs_fat.c

```diff
@@ -35,6 +35,9 @@
 #include "extmod/vfs_fat_file.h"
 #include "extmod/fsusermount.h"
 
+#include "timeutils.h"
+
```

Comment:
These 2 empty lines aren't needed, includes likely should just go together, with an empty line after them.